packages/@angular/cli/tasks/eject.ts

```diff
@@ -554,6 +554,16 @@ export default Task.extend({
         packageJson['scripts']['pree2e'] = pree2eNpmScript;
         packageJson['scripts']['e2e'] = 'protractor ./protractor.conf.js';
 
+        if (runTaskOptions.environment === 'production' && usesServiceWorker(project.root)
```

Comment:
I think this is `runTaskOptions.target` instead of `.enviroment`.

Reply:
Fixed.